### ISSUE TITLE
A11y_AzureDataStudio_Deploymenttemplateconfiguration_ScreenReader:Table markup is missing for "Aks -dev-test tab" 

### ISSUE THREAD

**Issue by V-sheman** [microsoft/azuredatastudio#7675]
GitHubTags: #A11y_AzureDataStudio;#A11yMAS; #A11yTCS;#SQL Azure Data Studio;#WCAG1.3.1;#WCAG4.2.3;#WCAG4.1.2;#WCAG4.2.1#DesktopApp;#Win32;#NVDA;
**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 32df727
Date: 2019-10-08T19:18:43.743Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
**Repro Steps:**
1.Install /Launch the application Azure data studio.
2.Navigate to connections and click on more action button.
3.Select “Deploy SQL Server big data cluster” from more action drop down.
4.Click on “Deploy SQL Server big data cluster”.
5. "Select the deployment option" screen gets open.
6.check the I accept check box and click on select button. 
7. Deployment template configuration screen gets open.
8. Start NVDA and navigate with the help of up and down arrow and observe the announcement.
**Actual:**
Issue 1. Table mark up is missing for the "Aks -dev-test tab" and the remaining tab on the screen.
Issue 2. Table name should be announced by the NVDA.
Issue 3. Row and column reference should be announced by the NVDA.
Issue 4. column header should be marked as a column header .
Expected:
Issue 1.There should be table mark up "Aks -dev-test tab" and the reaming tab on the screen.
Issue 2. Table name should be announced by the NVDA.
Issue 3. Row and column reference should be announced by the NVDA.
Issue 4. column header should be marked as a column header .
**User Impact:**
 In Aks -dev-test tab below content should be marked as table as   user will unable to understand the relation with the correspondent data.
Screen reader User will 

**Attachment for Reference:**

[7675_A11y_AzureDataStudio_Selectthedeploymentoption_ScreenReader4.pptx](https://github.com/microsoft/azuredatastudio/files/3717589/7675_A11y_AzureDataStudio_Selectthedeploymentoption_ScreenReader4.pptx)
Does this issue occur when all extensions are disabled?: Yes

**Comment by alanrenmsft** [comment]
this is not meant to be a table, according the definition of the roles, it is more like a button: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role, i am going to add a button role

**Comment by V-sheman** [comment]
Verified in Version: 1.15.0-insider (user setup)
Commit: 56695be14ac0160e7a4d23b7af939b98fdaaea45
Date: 2020-01-30T05:35:45.707Z
VS Code: 1.42.0
Electron: 6.1.6
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Windows_NT x64 10.0.18363  **t
[7675_fixed.pptx](https://github.com/microsoft/azuredatastudio/files/4132915/7675_fixed.pptx)
he bug but as you have said that you  are adding role but now issue is that NVDA focus is not going inside the radio button and description inside the radio button is not announced.**

**Comment by fsteffi** [comment]
#Azure Data Studio - New Features_win32;
